classes/02class/exercises/c02-network07/feernandobraga/README.md

```diff
@@ -0,0 +1,167 @@
+        }
+    ```
+
+    Renaming the routing table:
+
+    ```
+    aws ec2 create-tags --resources rtb-0e6c480035df5d6a7 --tags Key=Name,Value=devopsacademy-rt-public
+    ```
+
+  - ## Private
+
+    ```
+    aws ec2 create-route-table --vpc-id vpc-0c50c0a08b2a7dc7e
+
+    response:
+    {
+        "RouteTable": {
+            "Associations": [],
+            "PropagatingVgws": [],
+            "RouteTableId": "rtb-0e0fae42163550983",
+            "Routes": [
+                {
+                    "DestinationCidrBlock": "192.168.0.0/16",
+                    "GatewayId": "local",
+                    "Origin": "CreateRouteTable",
+                    "State": "active"
+                }
+            ],
+            "Tags": [],
+            "VpcId": "vpc-0c50c0a08b2a7dc7e",
+            "OwnerId": "667211700992"
+        }
+    }
+    ```
+
+    Renaming the routing table:
+
+    ```
+    aws ec2 create-tags --resources rtb-0e0fae42163550983 --tags Key=Name,Value=devopsacademy-rt-private
+    ```
+
+* Commands for associating the route tables with subnets:
+
+  - ## Public
+
+    ```
+    aws ec2 associate-route-table --route-table-id rtb-0e6c480035df5d6a7 --subnet-id subnet-059cef47dffda418b
+    response:
+    {
+        "AssociationId": "rtbassoc-05f98f635a8348a13",
+        "AssociationState": {
+            "State": "associated"
+        }
+    }
+
+    aws ec2 associate-route-table --route-table-id rtb-0e6c480035df5d6a7 --subnet-id subnet-065dc1da939f981f6
+    response:
+    {
+        "AssociationId": "rtbassoc-091f53415b8f2eff7",
+        "AssociationState": {
+            "State": "associated"
+        }
+    }
+
+    aws ec2 associate-route-table --route-table-id rtb-0e6c480035df5d6a7 --subnet-id subnet-009508e37b719dd77
+    response:
+    {
+        "AssociationId": "rtbassoc-08bbf333e59cf6273",
+        "AssociationState": {
+            "State": "associated"
+        }
+    }
+    ```
+
+- Commands for creating the following routes:
+
+| route table              | destination | target              |
+| ------------------------ | ----------- | ------------------- |
+| devopsacademy-rt-public  | 0.0.0.0     | `devopsacademy-igw` |
+| devopsacademy-rt-private | 0.0.0.0     | `devopsacademy-ngw` |
+
+```
+aws ec2 create-route --route-table-id rtb-0e6c480035df5d6a7 --destination-cidr-block 0.0.0.0/0 --gateway-id igw-06df7843381ed4767
+response:
+{
+    "Return": true
+}
+
+aws ec2 create-route --route-table-id rtb-0e0fae42163550983 --destination-cidr-block 0.0.0.0/0 --gateway-id nat-0b08ff466280436f1
+response:
+An error occurred (InvalidGatewayID.NotFound) when calling the CreateRoute operation: The gateway ID 'nat-0b08ff466280436f1' does not exist
+```
+
+- Answer the following questions:
+
+  - How did you accomplish both instructions?
+
+    ```
+    I was only able to accomplish the first one (public RT -> Internet gateway), since the NAT gateway was meant to be deleted right after exercise #06
```

Comment:
Sorry, it needs to be recreated again. I request deletion after the exercise for not forgetting it and getting a bill later. 

With the CLI command in hand from the previous exercise, it would be a matter of just executing it.